openrouteservice/src/main/java/org/heigit/ors/routing/RoutingProfile.java

```diff
@@ -1072,8 +1071,8 @@ private void setSpeedups(GHRequest req, boolean useCH, boolean useCore, boolean
         req.getHints().put(KEY_CORE_DISABLE, !useCore);
         req.getHints().put(KEY_LM_DISABLE, !useALT);
 
-        if(!useCH)
-            req.setAlgorithm(KEY_ASTARBI);
+        if (useCH)
+            req.setAlgorithm(Parameters.Algorithms.DIJKSTRA_BI);
```

Comment:
So we still want to fall back to plain dijkstra if CH is enabled?

Reply:
This should be fine, as CH uses Dijkstra rather than A*.  Once `useCH` has been set both CH is available for the given profile and weighting, and the request parameters are compatible with CH routing. So I don't think it can switch to a different algorithm afterwards.

Reply:
ok